pages/help_dialog.js

```diff
@@ -124,7 +124,17 @@ const HelpDialog = {
             });
           }
 
-          $$(descriptionElement, ".vimiumHelpDescription").textContent = command.description;
+          const MAX_LENGTH = 50;
+          const desiredOptionsLength = Math.max(0, MAX_LENGTH - command.description.length - 3);
+          const optionsTruncated = command.options.substring(0, desiredOptionsLength);
+          let ellipis = "";
+          if ((command.description.length + command.options.length) > MAX_LENGTH) {
+            ellipis = "...";
+            $$(descriptionElement, ".vimiumHelpDescription").title = command.options;
```

Comment:
It would be helpful to have a comment above this line saying "The non-ellipsized option list will be visible on-hover."